stunnel/init.sls

```diff
@@ -16,29 +16,29 @@ stunnel_package:
     - user: {{ stunnel.user }}
     - makedirs: True
 
-{% if grains['os_family'] == 'FreeBSD' -%}
 {{ stunnel.conf_dir }}/conf.d:
   file.directory:
-    - user: root
-    - group: wheel
-
-{{ stunnel.conf_dir }}/conf.d/pid.conf:
-  file.absent: []
+    - user: {{ stunnel.user }}
```

Comment:
There's a reason why only root can change the configuration: allowing stunnel to change it's own config is a security risk.